### ISSUE TITLE
Speed up TedPolicy training.

### ISSUE THREAD

**Issue by svenvanderburg** [PerfectFit-project/virtual-coach-server#215]
Training Rasa_Bot TedPolicy currently takes about an hour. Before it took only a few minutes. We should speed it up because it makes development really slow.

**Comment by svenvanderburg** [comment]
Some ideas:
* do not use augmentation, it contributes to accuracy, but slows down training, see: https://rasa.com/docs/rasa/policies/#data-augmentation
* Use a smaller `max_history`, it will decrease the model size
* Do not use `TEDPolicy` at all, just make use of AugmentedMemoizationPolicy and RulePolicy.

**Comment by svenvanderburg** [comment]
Some experiments:
## experiment 1
  - name: AugmentedMemoizationPolicy
    max_history: 14
  - name: TEDPolicy
    max_history: 5
    epochs: 50
    augmentation: 0

* training time: 10:44
* accuracy: stayed around 0.6 already after 25 epochs. Final acc=0.7
* Future self dialog fails at: 
* Nu krijg je twee zinnen te zien. Vul ze aan met wat het meest bij jou past en typ het nummer in de chat.
Ik zie mijzelf als... een roker (1) - een niet-roker (2) - iemand die stopt met roken (3)
Your input ->  1
Sorry, ik begrijp (nog) niet wat je bedoelt, kun je me het op een andere manier duidelijk maken?
Your input ->  2
Ik ben Kris. Ik ben je virtuele coach.
Ik ben hier om je te helpen stoppen met roken en lichamelijk actiever te worden.

**Comment by svenvanderburg** [comment]
Problem is not there on Walters and Neles Windows machines.
I managed to also get normal training times on my M1 Macbook pro with uninstalling tensorflow metal, see [that it can be slower](https://developer.apple.com/forums/thread/685623)

**Comment by svenvanderburg** [comment]
The issue on Boukes machine was that he was using tensorflow on his GPU.

On windows you can force using cpu by setting:
`set CUDA_VISIBLE_DEVICES=""`
